app/assets/javascripts/components/HomeLogin.jsx

```diff
@@ -0,0 +1,116 @@
+class HomeLogin extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: '',
+      password: '',
+      error: '',
+      hasError: false
+    }
+  };
+
+  _getToken() {
+    const token = document.getElementsByName("csrf-token")[0].getAttribute("content");
+    return token;
+  };
+
+  _handleKeydown(k) {
+    if (k.keyCode == 13) {
+      this._handleLogin();
+    }
+  }
+
+  handleChange(event) {
+    this.setState({ [$(event.target).attr("name")] : $(event.target).val() });
+  }
+
+  _handleLogin() {
+    var formFields = {
+      email: this.state.email,
+      password: this.state.password
+    }
+    Requester.post(`/api/sessions`, formFields).then(() => {
+      location.reload();
+    }, (e) => {
+      this.setState({ 
+        hasError: true,
+        error: 'Wrong email or password, please try again!'
+      });
+    })
+  };
+
+  showError() {
+    const Alert = ReactBootstrap.Alert
+    const { error } = this.state;
+    if (error != '') {
+      return (
+        <div className="wrong-login">
+        <Alert bsStyle="danger">
+          {error}
+        </Alert>
+        </div>
+      )
+    };
+  };
+
+  render() {
+    const Form = ReactBootstrap.Form,
+          FormGroup = ReactBootstrap.FormGroup,
+          FormControl = ReactBootstrap.FormControl,
+          Button = ReactBootstrap.Button;
+    return (
+      <div>
+      <Form>
+      <FormGroup>
+        <FormControl 
+        bsClass="form-control email-field" 
+        bsSize="lg"
+        type="email"
+        name="email"
+        placeholder="Email"
+        onChange={(e) => this.handleChange(e)}
```

Comment:
Do the thing above with setting it as a class property so we dont have to pass the arrow function in